### PR TITLE
6.5.2

### PR DESCRIPTION
**Pull request by eugbaranov**

### CODE REVIEW

File: tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,7 +1,19 @@
-﻿$packageName = "postman"
+$packageName = "postman"
 $fileType = "exe"
 $silentArgs = "-s"
-$url = "https://dl.pstmn.io/download/latest/win32"
-$url64 = "https://dl.pstmn.io/download/latest/win64"
+$url = "https://dl.pstmn.io/download/version/6.5.2/windows32"
+$url64 = "https://dl.pstmn.io/download/version/6.5.2/windows64"
```
Comment (kendaleiv) [inline tools/chocolateyInstall.ps1:5]: This package is intended to install the latest version available, even if the package itself is out of date. This reduces potential user frustration with installing an outdated version, only to have an update immediately available.

This strategy could work if the package was automatically updated, but I don't currently have anything like that setup. Ultimately, my main focus is delivering the latest.

Thanks for letting me know about the newer version though, https://chocolatey.org/packages/postman/6.5.2 is published.

**Comment by eugbaranov** [comment]
But doesn't this, in effect breaks Chocolatey versioning?
Let's say there is a bug in the latest version and I want to downgrade?

**Comment by kendaleiv** [comment]
I hadn't considered any downgrade scenario, was thinking everyone would likely want the latest. Is this something you've needed as a Postman user?

**Comment by eugbaranov** [comment]
Personally I prefer using the latest but every once in a while I had to downgrade to an older version, e.g. currently I have older version of `git` installed because the latest have a bug in it (Some Postman users had to downgrade: https://github.com/postmanlabs/postman-app-support/issues/4331). Also I tend to disable in-app automatic upgrades and just have a scheduled task to run `choco upgrade all` every week.

**Comment by kendaleiv** [comment]
That seems reasonable given the ability to disable automatic updates inside the app. fwiw, I've been in the habit of pinning some packages that upgrade themselves automatically to avoid installing upgrades twice.

**Pull request closed without merging**